src/GoodsList.tsx

```diff
@@ -1,16 +1,18 @@
-import React from 'react';
+import React, { memo } from 'react';
 import { Good } from './types/Good';
 
 type Props = {
-  goods: Good[]
+  goods: Good[];
 };
 
-export const GoodsList: React.FC<Props> = ({ goods }) => (
+const GoodsList: React.FC<Props> = ({ goods }) => (
```

Comment:
if you do not export native component, but only it's memoized version, name them respectively, e.g.
```suggestion
const NativeGoodsList: FC<Props> = ({ goods }) => (
```